tests/system/test_active_response/test_netsh_windows_command/test_firewall_status/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,32 @@
+- name: Enable/Disable firewall
+  hosts: windows-agent
+  tasks:
+
+    - name: Enable or disable firewall for Domain, Public and Private profiles
+      community.windows.win_firewall:
+        state: "{{ firewall_status }}"
+        profiles:
+          - Domain
+          - Private
+          - Public
+
+- name: Generate events
+  hosts: manager
+  tasks:
+
+    - name: Attempt a RDP brute force attack
+      shell: hydra -l {{ item }} -p invalid_password rdp://{{ hostvars['windows-agent']['ansible_host'] }}
+      loop:
+        - test_user
+
+- name: Copy log file to a tmp folder
+  hosts: windows-agent
+  tasks:
+
```

Comment:
We don't need any waiting time?

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3787/commits/e4ded61433bf9f440dd1336be139e4ecb01581ec)